README.md

```diff
@@ -37,13 +37,17 @@ The app can also operate in an offline mode in environments with limited interne
 ## Technology stack
 
 * [React Native](https://reactnative.dev/)
+    * Project is set with Latest version of react native:
+       * React native - 0.72.5
```

Comment:
Let's please change above two lines to "The project is configured to utilize React Native version 0.72.5."